config/locales/en.yml

```diff
@@ -745,6 +745,10 @@ en:
       ManageIQ::Providers::Kubernetes::ContainerManager::ContainerGroup:    Pod (Kubernetes)
       ManageIQ::Providers::Openshift::ContainerManager:                     Container Provider (Openshift)
       ManageIQ::Providers::OpenshiftEnterprise::ContainerManager:           Container Provider (Openshift Enterprise)
+      ManageIQ::Providers::Openstack::InfraManager::OrchestrationStack:     Orchestration Stack (OpenStack)
```

Comment:
maybe this one should be `OpenStack Infrastructure` or `OpenStack Undercloud` /cc @jonnyfiveiq